ubuntu1604/product.yml

```diff
@@ -10,3 +10,5 @@ init_system: "systemd"
 
 cpes:
   - "cpe:/o:canonical:ubuntu_linux:16.04"
+
+full_name: Ubuntu 1604
```

Comment:
16.04

I am also wondering if we should include the nickname for the release. I know a lot of ubuntu users know the nicknames better than release numbers.